packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -946,6 +946,14 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     return binding.takeException();
   }
 
+  /// Returns a list of all the accessibility announcements made by the Flutter
+  /// framework since the last time this function was called.
+  ///
+  /// See [TestWidgetsFlutterBinding.takeAnnouncements] for details.
```

Comment:
If the documentation can be identical, instead of pointing to another place, you can use dartdoc templates. Here's an example:

* Create a template: https://github.com/flutter/flutter/blob/9bdd50a5a49bee4e166faf6ac32d9ed8df2fb14b/packages/flutter/lib/src/cupertino/nav_bar.dart#L272
* Copy it to another place using a macro: https://github.com/flutter/flutter/blob/9bdd50a5a49bee4e166faf6ac32d9ed8df2fb14b/packages/flutter/lib/src/cupertino/nav_bar.dart#L631

Reply:
Cool feature! Thanks for pointing it out.